config/application.rb

```diff
@@ -28,6 +25,9 @@ class Application < Rails::Application
     config.plugins = [:dynamic_form]
 
     config.autoload_paths << Rails.root.join('lib')
+
+    # Do not halt callback chains when a callback returns false.
+    ActiveSupport.halt_callback_chains_on_return_false = false
   end
```

Comment:
@sonalkr132 Did you run `rails app:update`, this config will go to `config/initializers/new_framework_defaults.rb` after that.